apps/prairielearn/src/migrations/20250206202509_variants__authn_user_id__nonnull.sql

```diff
@@ -0,0 +1,33 @@
+-- We had a few old rows in the `variants` table where `authn_user_id` was NULL.
+-- We'll copy the values over from `variants.user_id` which is always non-NULL in the rows we care about.
+UPDATE variants
+SET
+  authn_user_id = user_id
+WHERE
+  authn_user_id IS NULL;
+
+-- Declare `variants.authn_user_id` as NOT NULL; it was always required.
+-- Use the approach described here to avoid a long table lock:
```

Comment:
I'm slightly nervous about this after the recent problem we had with a similar query, but I don't know what else to do?

Reply:
I'm also nervous, and also don't know what else to do about this 😬